app/main.py

```diff
@@ -2,4 +2,65 @@ class Cargo:
     def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(
+            self,
+            name: str,
+            weight: int,
+            coords: list[int, int] = None) -> None:
```

Comment:
If argument can be None, you should show it in type hint
```suggestion
            coords: list[int, int] | None = None) -> None:
```